PubNubUnity/Assets/PubNub/PlayModeTests/GrantToken.cs

```diff
@@ -10,22 +10,36 @@ public class PNGrantTokenResult{
         public string Token {get; set;}
     }
 
-    public class GrantTokenBuilder: PubNubNonSubBuilder<HereNowRequestBuilder, PNGrantTokenResult>, IPubNubNonSubscribeBuilder<GrantTokenBuilder, PNGrantTokenResult>{
+    public class GrantTokenBuilder: PubNubNonSubBuilder<GrantTokenBuilder, PNGrantTokenResult>, IPubNubNonSubscribeBuilder<GrantTokenBuilder, PNGrantTokenResult>{
```

Comment:
Shouldn't we remove Users and Spaces for set? We cannot allow set for users and spaces.

Reply:
this is internal only method, not exposed, only for integration tests.